components/match2/wikis/runeterra/match_summary.lua

```diff
@@ -0,0 +1,29 @@
+---
+-- @Liquipedia
+-- wiki=runeterra
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
+
+local CustomMatchSummary = {}
+
+---@param args table
+---@return Html
+function CustomMatchSummary.getByMatchId(args)
+	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {teamStyle = 'short'})
+end
+
+---@param date string
+---@param game MatchGroupUtilGame
+---@param gameIndex integer
+---@return Widget?
+function CustomMatchSummary.createGame(date, game, gameIndex)
+	return nil
+end
```

Comment:
intended to not display maps at all?
(totally fine with it for those kind of wikis, just making sure)

Reply:
Their current M1 never enters any map-level data